m2cgen/ast.py

```diff
@@ -107,3 +107,10 @@ def __init__(self, test, body, orelse):
     def __str__(self):
         args = ",".join([str(self.test), str(self.body), str(self.orelse)])
         return "IfExpr(" + args + ")"
+
+
+# Transparent Expressions
+
+class TransparentExpr(Expr):
```

Comment:
Not sure about the name. It doesn't really tell how to use it, but only suggests how it might be interpreted. Perhaps `LogicalBlockExpr`? Or `LogicalUnitExpr`?  Or even `DiscreteExpr`/`DiscreteUnitExpr`?